Demo/Demo/SwiftUIComponents/SwiftUIPaymentButtonDemo.swift

```diff
@@ -74,18 +74,18 @@ struct SwiftUIPaymentButtonDemo: View {
 
                 switch selectedFunding {
                 case .payPal:
-                    PayPalButton(color: PayPalButton.Color.allCases()[colorsIndex], edges: selectedEdge, size: selectedSize).id(buttonId)
+                    PayPalButton(color: UIPayPalButton.Color.allCases()[colorsIndex], edges: selectedEdge, size: selectedSize).id(buttonId)
```

Comment:
It feels a little unintuitive that SwiftUI refers to the UIKit button to fetch colors. Would merchants do something similar?